### ISSUE TITLE
How to perform multi-scale test

### ISSUE THREAD

**Issue by lxtGH** [open-mmlab/mmdetection#1165]
Hi, I want to use the trained detector to detect car and people in images and save it into json file.
How to perform the multi-scale test using your trained model on validation set. Any refernce code ??  How to perform the multi-scale test on test set to submit to the server ?

**Comment by hellock** [comment]
#135

**Comment by hellock** [comment]
For your second question, the output json file follows the standard format of COCO results, which is irrelevant to single-scale or multi-scale testing.

**Comment by VectorYoung** [comment]
Hi, I am new to object detection. I didn't quite understand the multi-scale testing. Does it mean first go through the test data in one scale, then go through test data in another scale, finally average the AP in different scales as the final AP?

**Comment by lxtGH** [comment]
@hellock Hi!! Thanks for your reply. I also have a question. Could you share choice of scales when performing multi-scale test ?

**Comment by hellock** [comment]
There is not a optimal choice for scales and that depends on the datasets and methods. Many papers present the scales they adopt for multi-scale testing and you may refer to them.

**Comment by lxtGH** [comment]
Thank you !
